muxrpc/handlers/tunnel/server/plugin.go

```diff
@@ -59,6 +59,7 @@ func (h *Handler) RegisterRoom(mux typemux.HandlerMux) {
 	mux.RegisterAsync(append(namespace, "ping"), typemux.AsyncFunc(h.ping))
 
 	mux.RegisterSource(append(namespace, "attendants"), typemux.SourceFunc(h.attendants))
+	mux.RegisterSource(append(namespace, "members"), typemux.SourceFunc(h.members))
```

Comment:
Unsure...

```suggestion
	mux.RegisterSource(append(namespace, "members"), typemux.SourceFunc(h.membersdb))
```

Reply:
The alternative would be to rename all handler functions to `...Handler` eg. `membersHandler` for clarity.